### PR TITLE
Issue 4 user can delete their entries

### PR DESCRIPTION
**Pull request by james-aguirre**
Screenshot 2023-04-17 at 11 37 00 AM
Screenshot 2023-04-17 at 11 36 33 AM

### CODE REVIEW

**Comment by james-aguirre** [comment]
Kapture 2023-04-17 at 13 48 06

**Comment by james-aguirre** [comment]
Kapture 2023-04-18 at 11 40 31

File: css/styles.css [hunk 1]
```diff
@@ -121,5 +122,67 @@ h3 {
...
+  left: 50%;
+  margin-left: -13rem;
+  top: 50%;
+  margin-top: -7rem;
+  height: 8.5rem;
+  width: 26rem;
+  background-color: white;
+  padding-left: 1rem;
+  text-align: center;
+  font-size: 1rem;
```
Comment (shawnkost) [inline css/styles.css:152]: There shouldn't be a need for using such large `rem` values, if you're using such large values, you're probably doing something "hacky" to get this to work. You shouldn't be needing to use width & height alongside offset values. You should be using one or the other. Revisiting `javascript-modal` would be a good exercise for this

Your modal should be split into 2. The outer div / parent div controls where it appears on the page. And then inside of that, you should have another div where all the content belongs.

File: js/main.js [hunk 2]
```diff
@@ -97,11 +112,12 @@ document.addEventListener('DOMContentLoaded', function (event) {
   for (let i = 0; i < data.entries.length; i++) {
     $list.appendChild(renderEntry(data.entries[i]));
   }
-  viewSwap(data.view);
+  viewSwap('entries');
```
Comment (shawnkost) [inline js/main.js:115]: This should be `data.view`. You don't want to swap to a specific page, you want to utilize whatever was stored on your last refresh

File: js/main.js [hunk 3]
```diff
@@ -115,12 +131,70 @@ $anchorTag.addEventListener('click', function () {
...
+  $modal.classList.remove('hidden');
+  $body.classList.add('overlay');
+  $notes.classList.add('dark');
+  $title.classList.add('dark');
+  $photo.classList.add('opacity');
+  $img.classList.add('dark');
+  $mainHeader.classList.add('opacity');
+  $deleteButton.classList.add('opacity');
+  $saveButton.classList.add('opacity');
+  $formHeader.classList.add('opacity');
```
Comment (shawnkost) [inline js/main.js:153]: There shouldn't be a need to give so many different elements a class of `opacity` or dark to apply the filter on them. You should just have a background color on your `modal` container, and that will impact everything

File: js/main.js [hunk 4]
```diff
@@ -115,12 +131,70 @@ $anchorTag.addEventListener('click', function () {
...
+  $body.classList.remove('overlay');
+  $notes.classList.remove('dark');
+  $title.classList.remove('dark');
+  $img.classList.remove('dark');
+  $photo.classList.remove('opacity');
+  $mainHeader.classList.remove('opacity');
+  $deleteButton.classList.remove('opacity');
+  $saveButton.classList.remove('opacity');
+  $formHeader.classList.remove('opacity');
+}
```
Comment (shawnkost) [inline js/main.js:169]: Same note as above ^

File: js/main.js [hunk 5]
```diff
@@ -115,12 +131,70 @@ $anchorTag.addEventListener('click', function () {
...
+  toggleNoEntries();
+  $modal.classList.add('hidden');
+  $body.classList.remove('overlay');
+  $notes.classList.remove('dark');
+  $title.classList.remove('dark');
+  $img.classList.remove('dark');
+  $photo.classList.remove('opacity');
+  $mainHeader.classList.remove('opacity');
+  $deleteButton.classList.remove('opacity');
+  $saveButton.classList.remove('opacity');
```
Comment (shawnkost) [inline js/main.js:193]: Same note above ^

**Review by shawnkost — CHANGES_REQUESTED** [review]
A few changes here James!

**Comment by james-aguirre** [comment]
Screenshot 2023-04-20 at 10 28 56 AM

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**